msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/sdk/Constants.java

```diff
@@ -32,4 +32,6 @@ public class Constants {
     public static final String TOKEN_URL_PATH = "p";
 
     public static final String TOKEN_HTTP_METHOD = "m";
+
+    public static final boolean IS_EXPECTING_SECOND_SPEED_BUMP = false;
```

Comment:
Is this a cross cloud UX?

Reply:
This was briefly showing up for a couple tests right before last month's release process. Now seems to have gone away. Yes, it's cross cloud related